src/main/scripts/src/app/workspace/project/project-settings.component.html

```diff
@@ -1,13 +1,95 @@
-<ul class="nav nav-tabs">
-  <li role="presentation" class="nav-item">
-    <a routerLink="settings" routerLinkActive="active" class="nav-link">
-      Basic Settings
+<div class="d-flex flex-row justify-content-start">
+  <a class="fa fa-arrow-left " [routerLink]="['/workspace']  "></a>
+  <i class="color-class fa-fw {{ project.icon }} orgIcon " [style.color]="project.color"
+     data-toggle="dropdown"></i>
+  <div class="dropdown-menu">
+    <color-picker class="dropdown-item" [(color)]="project.color"
+                  (selectedChange)="updateProject()"></color-picker>
+    <div class="dropdown-divider"></div>
+    <icon-picker class="dropdown-item" [(icon)]="project.icon"
+                 (selectedChange)="updateProject()"></icon-picker>
+  </div>
+
+  <input class="color-class name" [(ngModel)]="project.name" (blur)="updateProject()"
+         [style.color]="project.color" title="Edit name of the project">
+
+  <div class="float-right document-count color-class ml-auto">
+    <a class="count" title="Number of collections in this project" [style.color]="project.color">
```

Comment:
These should redirect since they are `<a>`

`<a class="count" title="Number of collections in this project" [style.color]="project.color">` ->
```
<a class="count" 
    title="Number of collections in this project" 
    [routerLink]="[workspacePath(), 'collections']" 
    [style.color]="project.color">
```